plugins/tiddlywiki/menubar/config.tid

```diff
@@ -28,7 +30,7 @@ The breakpoint position between narrow and wide screens. Should include CSS unit
 
 !! Contents Tag
 
-The tag for the ~TableOfContents used in the Contents dropdown
```

Comment:
Would it be worth including text editor widgets that allow the caption and description fields to be directly inspected/edited?

Reply:
Done in my next commit